core/src/main/python/wlsdeploy/tool/deploy/applications_deployer.py

```diff
@@ -272,9 +280,6 @@ def __online_deploy_apps_and_libs(self, base_location):
         self.__deploy_model_libraries(model_shared_libraries, lib_location)
         self.__deploy_model_applications(model_applications, app_location, deployed_app_list)
 
-        for app in redeploy_app_list:
-            self.__redeploy_app(app)
```

Comment:
Does the __redeploy_app method need to be removed also?  Doesn't appear to be used anywhere else.

Reply:
Yes, I will remove it